caikit_nlp/toolkit/task_specific_utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Local
+from ..data_model import ClassificationTrainRecord, GenerationTrainRecord
+
+
+def convert_to_generation_record(TrainRecord):
```

Comment:
nit: Can you please add docstring for this function explaining the intent

Reply:
since `TrainRecord` is a variable here, having it in camel case is a bit confusing. Can we change this to `snake_case` ?

Reply:
ya sorry, my bad